documentation/src/docs/deployment/manual-install.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Manual Installation
```

Comment:
Same problem with titles: "Installing Pravega" appears super large and is not in the right side index.

Reply:
fixed title sizes and right-side index